GenderPayGap.WebUI/Models/Admin/AdminOrganisationCompanyNumberViewModel.cs

```diff
@@ -23,11 +23,10 @@ public class AdminOrganisationCompanyNumberViewModel
         public bool ChangeOrRemoveRequired => CurrentPage is AdminOrganisationCompanyNumberViewModelCurrentPage.OfferChangeOrRemove;
         [GovUkValidateRequiredIf(IsRequiredPropertyName = nameof(ReasonRequired), 
             ErrorMessageIfMissing = "Please enter a reason for this change")]
-        [GovUkValidateCharacterCount(MaxCharacters = 250)]
+        [GovUkValidateCharacterCount(MaxCharacters = 250, NameAtStartOfSentence = "Reason", NameWithinSentence = "Reason")]
         public string Reason { get; set; }
 
-        public bool ReasonRequired => CurrentPage is AdminOrganisationCompanyNumberViewModelCurrentPage.Remove ||
-                                      CurrentPage is AdminOrganisationCompanyNumberViewModelCurrentPage.ConfirmNew;
+        public bool ReasonRequired => CurrentPage is AdminOrganisationCompanyNumberViewModelCurrentPage.Remove;

```

Comment:
Why has this changed?

Reply:
I double checked how the code was before.
Remove action:
`viewModel.ParseAndValidateParameters(Request, m => m.Reason);

            if (viewModel.HasAnyErrors())
            {
                viewModel.Organisation = organisation;
                return View("RemoveOrganisationCompanyNumber", viewModel);
            }
...`

Change action:
`viewModel.ParseAndValidateParameters(Request, m => m.NewCompanyNumber);
            viewModel.Organisation = organisation;

            if (viewModel.HasAnyErrors())
            {
                return View("ChangeOrganisationCompanyNumber", viewModel);
            }
...`

Change apparently did not require a Reason so I reverted that

Reply:
Cool, thanks for checking.